doc/sphinx/proofs/writing-proofs/inductive.rst

```diff
@@ -244,26 +249,26 @@ analysis on inductive or co-inductive objects (see :ref:`inductive-definitions`)
    hypotheses are added to the local context using names whose prefix
    is **IH**.
 
-   There are particular cases:
+   There are several cases:
 
-   + If term is an identifier :n:`@ident` denoting a quantified variable of the
-     conclusion of the goal, then inductionident behaves as :n:`intros until
-     @ident; induction @ident`. If :n:`@ident` is not anymore dependent in the
+   + If :n:`@one_term` is an identifier :n:`@ident` denoting a quantified variable of the
+     conclusion of the goal, then inductionident behaves like :n:`intros until
+     @ident; induction @ident`. If :n:`@ident` is no longer dependent in the
      goal after application of :n:`induction`, it is erased (to avoid erasure,
      use parentheses, as in :n:`induction (@ident)`).
```

Comment:
I thought this only worked for hypotheses.  Does this trick really work for quantified variables of the goal too?  If so, induction is more magic than I thought it was.